src/components/backend-ai-storage-list.ts

```diff
@@ -2460,6 +2519,8 @@ export default class BackendAiStorageList extends BackendAIPage {
             globalThis.backendaiclient._config.maxFileUploadSize;
           this.directoryBasedUsage =
             globalThis.backendaiclient._config.directoryBasedUsage;
+          this._isDirectorySizeVisible =
+            globalThis.backendaiclient._config.isDirectorySizeVisible;
```

Comment:
You have to add this code to `else` block (line number 2544).

Reply:
Without this modification, there is a bug where the folder size is displayed. To reproduce this issue:

- Visit another page first and then click the `data & folder` menu
- Click on the folder name
- You can see the folder size